codec-http/src/main/java/io/netty/handler/codec/rtsp/RtspEncoder.java

```diff
@@ -29,15 +26,14 @@
 import io.netty.util.CharsetUtil;
 import io.netty.util.internal.StringUtil;
 
+import static io.netty.handler.codec.http.HttpConstants.*;
+
 /**
  * Encodes an RTSP message represented in {@link HttpMessage} or an {@link HttpContent} into
  * a {@link ByteBuf}.
  */
 public class RtspEncoder extends HttpObjectEncoder<HttpMessage> {
-    /**
-     * Constant for CRLF.
-     */
-    private static final byte[] CRLF = {CR, LF};
+    private static final int CRLF_SHORT = (CR << 8) | LF;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) "CRLF_SHORT" is the name of a field in "HttpObjectEncoder". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS2387)